packages/gluestick/src/config/webpack/webpack.config.server.js

```diff
@@ -57,6 +58,12 @@ module.exports = (
   );
   config.resolve.alias['plugins-config-path'] =
     gluestickConfig.pluginsConfigPath;
+  config.resolve.alias['react-universal-component/server'] = path.join(
```

Comment:
This prevents a duplicate package from being imported. Normally that's annoying and a bit expensive, but in this case it causes the whole system to break down, as there's a single cache for both server and client.

There may be a better way to do this.